### PR TITLE
feat: update operator to use latest kubebuilder layout

### PR DESCRIPTION
**Pull request by dejanzele**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: config/crd/bases/tests.testkube.io_testsuites.yaml [hunk 1]
```diff
@@ -656,6 +668,18 @@ spec:
                   pvcTemplateReference:
                     description: name of the template resource
                     type: string
+                  runningContext:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_testsuites.yaml:671]: we don't use in Cd

File: config/crd/bases/tests.testkube.io_testsuites.yaml [hunk 2]
```diff
@@ -356,6 +356,18 @@ spec:
                     description: test kubernetes namespace (\"testkube\" when not
                       set)
                     type: string
+                  runningContext:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_testsuites.yaml:359]: we don't use it in CRD

File: config/crd/bases/tests.testkube.io_testsources.yaml [hunk 3]
```diff
@@ -70,6 +70,9 @@ spec:
                       name:
                         description: object name
                         type: string
+                      namespace:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_testsources.yaml:73]: we don't use it in CRD

File: config/crd/bases/executor.testkube.io_executors.yaml [hunk 4]
```diff
@@ -51,9 +51,7 @@ spec:
                   enum:
                   - string
                   - file-uri
-                  # Deprecated: use git instead
```
Comment (vsukhin) [inline config/crd/bases/executor.testkube.io_executors.yaml:54]: we need it for CRD

File: Dockerfile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-ARG ALPINE_IMAGE
+ARG ALPINE_IMAGE=alpine:3.18.3
```
Comment (vsukhin) [inline Dockerfile:1]: why is it hardcoded?

File: config/crd/bases/tests.testkube.io_tests.yaml [hunk 6]
```diff
@@ -388,6 +388,9 @@ spec:
                           name:
                             description: object name
                             type: string
+                          namespace:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_tests.yaml:391]: we don't use it for CRD

File: config/crd/bases/tests.testkube.io_tests.yaml [hunk 7]
```diff
@@ -408,6 +411,9 @@ spec:
...
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_tests.yaml:414]: we don't use it for CRD

File: config/crd/bases/tests.testkube.io_tests.yaml [hunk 8]
```diff
@@ -612,6 +618,18 @@ spec:
                   pvcTemplateReference:
                     description: name of the template resource
                     type: string
+                  runningContext:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_tests.yaml:621]: we don't use it for CRD

File: Dockerfile (hunk 5 above)
Comment (dejanzele) [reply Dockerfile]: this is to add default in case it is not provided, it is easier to build locally to test

**Pull request merged**